### PR TITLE
Feature: Upgrade to eslint-plugin 8.x

### LINKED ISSUE #132: Feature: Upgrade to eslint-plugin 8.x
Dependabot has been trying to update to @typescript-eslint/eslint-plugin from 7.0.0 to 8.x for a while now but the builds always fail. I haven't had a chance to look at this.

This would be a great first issue for anyone interested in TypeScript and ESLint.

Here's the latest failure, https://github.com/nickytonline/collabiano/pull/131

### PR DESCRIPTION
**Pull request by lmac-1**
## Description
This PR upgrades `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` to 8.4.0  

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] 📦 Chore
- [ ] ⏩ Revert

## Related Tickets & Documents
Fixes #132

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📜 CONTRIBUTING.md
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
At first:
unsure

Now:
happy

### CODE REVIEW

**Comment by lmac-1** [comment]
Let me know if I've missed anything on the PR or if you need something else :) thanks and have a great day!

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package.json
+package-lock.json
```
Comment (nickytonline) [inline .eslintignore:2]: nit: add a new line at the end of the file. Prettier usually handles this. 🤔

**Review by nickytonline — APPROVED** [review]
Thanks @lmac-1! 🚢

File: .eslintignore (hunk 1 above)
Comment (lmac-1) [reply .eslintignore]: Done ✅

**Pull request merged**